scipy/optimize/tests/test_linprog.py

```diff
@@ -1646,32 +1647,49 @@ def test_sensitivies(self):
         c, A_ub, b_ub, A_eq, b_eq, bounds = very_random_gen(seed=0)
         res = linprog(c, A_ub=A_ub, b_ub=b_ub, A_eq=A_eq, b_eq=b_eq,
                       bounds=bounds, method=self.method, options=self.options)
+        lb, ub = bounds.T
+
+        # sensitivity w.r.t. b_ub
+        def f_bub(x):
+            return linprog(c, A_ub, x, A_eq, b_eq, bounds,
```

Comment:
I like how you can see the `x` move around in each variation of the objective function -- it makes it a lot clearer than my terse perturbations